README.md

```diff
@@ -327,8 +327,8 @@ The example below shows how you might convert and intersect two arbitrary SVG el
 ```javascript
 const {SvgShapes, Intersection} = require("kld-intersections");
 
-const shape1 = SvgShapes.element(document.getElementById("a"));
-const shape2 = SvgShapes.element(document.getElementById("b"));
+const shape1 = SvgShapes.element(document.querySelector("a"));
```

Comment:
I believe this changes the semantics. My version grabs an element by id. Your version looks for the first "a" element in the document.

Reply:
Unfortunately, there is no all-purpose linter yet for absent-mindedness. Fixed (and rebased)! :)